samples/springboot3/function-sample-aws-serverless-web-native/pom.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.2.0-SNAPSHOT</version>
+		<relativePath /> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>oz.native.sample</groupId>
+	<artifactId>function-sample-aws-serverless-web-native</artifactId>
+	<version>0.0.1-SNAPSHOT</version><!-- @releaser:version-check-off -->
+	<name>function-sample-aws-serverless-web-native</name>
+	<description>Sample of AWS with Spring Native</description>
+	<properties>
+		<java.version>21</java.version>
+		<spring-cloud.version>2022.0.5-SNAPSHOT</spring-cloud.version>
+	</properties>
+	<dependencies>
+		<dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+            <exclusions>
+                <exclusion>
```

Comment:
@deki We should also do this for the other samples / workshops when using the starter. Its much cleaner than the filter in the shade plugin.

Reply:
actually i'll remove this all together since exclusion is already included in function library